rocAL/rocAL_pybind/amd/rocal/plugin/generic.py

```diff
@@ -139,12 +101,12 @@ def __next__(self):
                     for i in range(self.bs):
                         img = (self.out)
                         draw_patches(img[i], i, 0)
-                self.loader.getImageLabels(self.labels)
+                self.labels = self.loader.rocalGetImageLabels()
                 if self.device == "cpu":
-                    self.labels_tensor = self.labels.astype(dtype=np.int_)
+                    self.labels_tensor = self.labels.astype(dtype = np.int_)
                 else:
                     with cp.cuda.Device(device=self.device_id):
-                        self.labels_tensor = self.labels.astype(dtype=cp.int_)
+                        self.labels_tensor = self.labels.astype(dtype = cp.int_)
```

Comment:
If adding spaces before and after = is planned, do it consistently for all parameters in all files. Check L196

Reply:
corrected the spacing in both files generic.py & pytorch.py